doc/releases/changelog-dev.md

```diff
@@ -316,39 +363,11 @@
   system is disabled.
   [(#8532)](https://github.com/PennyLaneAI/pennylane/pull/8532)
 
-* A new decomposition has been added to :class:`pennylane.Toffoli`. This decomposition uses one
-  work wire and :class:`pennylane.TemporaryAND` operators to reduce the resources needed.
-  [(#8549)](https://github.com/PennyLaneAI/pennylane/pull/8549)
-
-* The :func:`~pennylane.pauli_decompose` now supports decomposing scipy's sparse matrices,
-  allowing for efficient decomposition of large matrices that cannot fit in memory when written as
-  dense arrays.
-  [(#8612)](https://github.com/PennyLaneAI/pennylane/pull/8612)
-  
-* A decomposition has been added to the adjoint of :class:`pennylane.TemporaryAND`. This decomposition relies on mid-circuit measurments and does not require any T gates.
-  [(#8633)](https://github.com/PennyLaneAI/pennylane/pull/8633)
-
-* The graph-based decomposition system now supports decomposition rules that contains mid-circuit measurements.
-  [(#8079)](https://github.com/PennyLaneAI/pennylane/pull/8079)
-
 * The `~pennylane.estimator.compact_hamiltonian.CDFHamiltonian`, `~pennylane.estimator.compact_hamiltonian.THCHamiltonian`,
   `~pennylane.estimator.compact_hamiltonian.VibrationalHamiltonian`, and `~pennylane.estimator.compact_hamiltonian.VibronicHamiltonian`
   classes were modified to take the 1-norm of the Hamiltonian as an optional argument.
   [(#8697)](https://github.com/PennyLaneAI/pennylane/pull/8697)
 
-* New decomposition rules that decompose to :class:`~.PauliRot` are added for the following operators.
-  [(#8700)](https://github.com/PennyLaneAI/pennylane/pull/8700)
-  [(#8704)](https://github.com/PennyLaneAI/pennylane/pull/8704)
-
-  - :class:`~.CRX`, :class:`~.CRY`, :class:`~.CRZ`
-  - :class:`~.ControlledPhaseShift`
-  - :class:`~.IsingXX`, :class:`~.IsingYY`, :class:`~.IsingZZ`
-  - :class:`~.PSWAP`
-  - :class:`~.RX`, :class:`~.RY`, :class:`~.RZ`
-  - :class:`~.SingleExcitation`, :class:`~.DoubleExcitation`
-  - :class:`~.SWAP`, :class:`~.ISWAP`, :class:`~.SISWAP`
-  - :class:`~.CY`, :class:`~.CZ`, :class:`~.CSWAP`, :class:`~.CNOT`, :class:`~.Toffoli`
-
 <h3>Breaking changes 💔</h3>
```

Comment:
Can't comment on this line exactly, but this line: 

> argnum has been renamed argnums for qml.grad, qml.jacobian, qml.jvp and qml.vjp. [(#8496)](https://github.com/PennyLaneAI/pennylane/pull/8496) [(#8481)](https://github.com/PennyLaneAI/pennylane/pull/8481)

can go in Breaking changes instead of deprecations :)

Reply:
Also 

>qml.cond, the QNode, transforms, qml.grad, and qml.jacobian no longer treat all keyword arguments as static arguments. They are instead treated as dynamic, numerical inputs, matching the behaviour of Jax and Catalyst. [(#8290)](https://github.com/PennyLaneAI/pennylane/pull/8290)

this can go in breaking changes

Reply:
> The constant to convert the length unit Bohr to Angstrom in qml.qchem is updated to use scipy constants. [(#8537)](https://github.com/PennyLaneAI/pennylane/pull/8537)

This can go in Other Improvements :)

Reply:
> The [GeneralizedAmplitudeDamping](https://xanaduai-pennylane--8766.com.readthedocs.build/en/8766/code/api/pennylane.GeneralizedAmplitudeDamping.html) error channel method has been updated to match the literature convention for the definition of the Kraus matrices. [(#8707)](https://github.com/PennyLaneAI/pennylane/pull/8707)

This can go in the Documentation section :)